src/Presentation/Presenters/MembershipApplicationConfirmationHtmlPresenter.php

```diff
@@ -74,4 +76,8 @@ private function getBankDataArguments( PaymentMethod $payment ): array {
 		return [];
 	}
 
+	private function mapApplicationStatusToTranslatablePaymentStatus( bool $isConfirmed ): string {
+		return $isConfirmed ? 'B' : 'X';
```

Comment:
I don't like using short "magic strings". If you proceed with this PR, please put them in constants with meaningful names.